### ISSUE TITLE
Register a new account

### ISSUE THREAD

**Issue by dessalines** [dessalines/jerboa#29]

**Comment by Schoolkid1** [comment]
How do people up to new create accounts?

**Comment by abeorch** [comment]
Rather thas an App based server selection and signup process would it be better to send the user to https://join-lemmy.org/ to review server characteristcs and select a server to signup on?

**Comment by abeorch** [comment]
> Just look at how lemmy-ui's signup screen looks. I'd see it as:
> 
>     * By default, show the login screen, but have a button at the bottom for create an account.
> 
>     * Have a link to https://join-lemmy.org/?showJoinModal=true to help them pick.
> 
>     * Have a dropdown just like the login one
> 
>     * After a server is selected, load its signup form.
> 
>     * Let them sign up.

Yeah .. its three options 1) Login 2) Choose which server to create an account on  3) Create an account on a selected server -  I think in that order 

There is a subsquent step after 2) and 3) where after creating an account they need to come back and login - I guess that's something that needs to be handled by the success page of the Server signup form - allowing the user to Login using Jerboa or other App

**Comment by MV-GH** [comment]
I imagined a bit more informative, when selecting an instance. It will show quite some information about it.

- account creation allow
- downvote enabled
- federation disabled: instances
- approval required
- language?
- Option to see instance description?
- email required?
- not robot check
...

Then you can select continue and sign up process

**Comment by abeorch** [comment]
If that information is availabe its all included in the Join-lemmy interface and ultimately to create an account you need to land on the servers's signup page so using the existing interface makes sense to me.

**Comment by dessalines** [comment]
One thing we can't really do (unless we pull and convert [instance definitions](https://github.com/LemmyNet/joinlemmy-site/blob/main/src/shared/components/instances-definitions.ts) from the join-lemmy repo), is help people pick an instance.

**Comment by abeorch** [comment]
Agree that is their decision when they head over to Join-lemmy (which I think its redundant to include in the app itself since its only a transit point to the server subscribe page. - I'm interested in what the success page of the server subscribe page looks like. I need to get the skills to spin up a test lemmy server to figure that out.
